### PR TITLE
[events] Changes to event queries

### PR DESCRIPTION
**Pull request by damirka**
1. Increases the limit on event fetches (100 -> 1000);
2. Changes DESC to ASC following the natural order of txs (from old to new);

### CODE REVIEW

File: crates/sui-storage/src/event_store/sql.rs [hunk 1]
```diff
@@ -329,28 +329,28 @@ impl From<SqliteRow> for StoredEvent {
 }
 
 const TS_QUERY: &str =
-    "SELECT * FROM events WHERE timestamp >= ? AND timestamp < ? ORDER BY timestamp DESC LIMIT ?";
+    "SELECT * FROM events WHERE timestamp >= ? AND timestamp < ? ORDER BY timestamp ASC LIMIT ?";
```
Comment (velvia) [inline crates/sui-storage/src/event_store/sql.rs:332]: I'm fine changing these to ASC if most users would prefer increasing time order.   Can you please update the RustDoc in `event_store/mod.rs` in the Trait though?  Thanks

**Review by velvia — COMMENTED** [review]
I'm fine with the event side changes, except for adding / changing the RustDoc in the EventStore Trait, as that specifies decreasing time order.

File: crates/sui-storage/src/event_store/sql.rs (hunk 1 above)
Comment (damirka) [reply crates/sui-storage/src/event_store/sql.rs:332]: Sure can!

Comment (damirka) [reply crates/sui-storage/src/event_store/sql.rs:332]: Done!

**Review by velvia — APPROVED** [review]
Thanks @damirka .... I'm approving, but this is contingent on your perspective that ascending is better for users.   @patrickkuo and @longbowlu should comment as well, because Patrick is working on some kind of pagination thing, and this might affect his work?

**Review by patrickkuo — APPROVED** [review]
LGTM!

For my upcoming pagination changes, first I am merging all getEvents* methods and use a EventQuery arg instead of separate methods, and then add a cursor based pagination to the new getEvents method. It is quite disruptive but should be easy to migrate.

**Comment by damirka** [comment]
Merging this buddy in after tests pass.

**Pull request merged**